docs/docusaurus/docs/cloud/deploy_gx_agent.md

```diff
@@ -12,7 +12,7 @@ import Tabs from '@theme/Tabs';
 
 To use GX Cloud features and functionality, you need to deploy the GX Agent. The GX Agent is an intermediary between GX Cloud and your organization's data stores. GX Cloud does not connect directly to your data, all data access occurs within the GX Agent. GX Cloud sends jobs to the GX Agent, the GX Agent executes these jobs against your data, and then sends the job results to GX Cloud.
 
-The GX Agent is typically deployed in your organization's deployment environment, for example, in a development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows. To learn more about the GX Agent and deployment patterns, see [About GX Cloud](./about_gx.md).
+To try GX Cloud, you use a [self-hosted deployment](./about_gx#self-hosted-deployment-pattern) to run the GX Agent with Docker, connect the GX Agent to your target Data Sources, and use the GX Cloud web UI to define your Data Assets, create Expectations, and run Validations. When you've finished testing GX Cloud, you move to an [org-hosted deployment](./about_gx.md#org-hosted-deployment-pattern) and deploy the GX Agent to your organization's development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows. To learn more about the GX Agent and deployment patterns, see [About GX Cloud](./about_gx.md).
```

Comment:
The term "try GX Cloud" has been confusing to me in the past– I don't associate "try" with a temporary or test setup, I associate it with getting started. 

If we want to emphasize the ephemeral local setup, maybe we say "To test GX Cloud, you may use a self-hosted deployment...."? What do you think @kwcanuck ?